hazelcast/src/test/java/com/hazelcast/jet/core/ExecutionLifecycleTest.java

```diff
@@ -184,6 +189,22 @@ public void when_pmsInitThrows_then_jobFails() {
         assertJobFailed(job, MOCK_ERROR);
     }
 
+    //ProcessorMetaSupplier throws NonSerializable exception from init()
+    @Test
+    public void when_pmsInitThrowsNonSerializable_then_jobFails() {
+        // Given
+        SupplierEx<ProcessorSupplier> supplier = ()->  new MockPS(MockP::new, MEMBER_COUNT);
+        DAG dag = new DAG().vertex(new Vertex("test",
+                new TestProcessors.NonSerializingMockPMS(supplier)));
```

Comment:
Personally I'd import static those inner classes

Reply:
MockPMS is used